joomla/40-backend-template
# support pattern validation

Pull Request for Issue # .

### Summary of Changes
HTML5 NATIVE validation is done through `pattern="something"`
The validate.js is adjusted so if pattern tag exists will take precedence else we fall back to the rules

### Testing Instructions
- try adding any of the patterns to a text, email, etc field and see if this works
- use http://html5pattern.com/Names for reference

Add `pattern="[a-zA-Z]+"` after https://github.com/joomla/40-backend-template/blob/master/administrator/components/com_content/models/forms/article.xml#L13
Create a new article and try to submit withou title and with title = (87587676) (or any other number)

![screen shot 2016-12-05 at 16 04 38](https://cloud.githubusercontent.com/assets/3889375/20887733/8f1e0ee2-bb04-11e6-897d-8824042c4231.png)


### TODO
- Add patterns to all the forms all over the admin area (form.xml)

### Documentation Changes Required
